modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-web/src/main/resources/META-INF/resources/my_account/setup_user_account.jsp

```diff
@@ -27,6 +28,7 @@ SetupMFAChecker setupMFAChecker = (SetupMFAChecker)request.getAttribute(SetupMFA
 
 <aui:form action="<%= actionURL %>" cssClass="portlet-users-admin-edit-user" data-senna-off="true" method="post" name="fm">
 	<aui:input name="redirect" type="hidden" value="<%= currentURL %>" />
```

Comment:
This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation: